stanford_profile.post_update.php

```diff
@@ -46,3 +49,57 @@ function stanford_profile_post_update_8014() {
   ]);
 
 }
+
+/**
+ * Create blocks for sites with custom themes that were added to stanford_basic.
+ */
+function stanford_profile_post_update_8015() {
```

Comment:
This was done in another PR wasn't it?

Reply:
It went into this branch.

Reply:
Suppose I could have pushed it into the main branch but I was building the deployments from this so it was easier